src/Apps/Collect/Components/Collection/Header/index.tsx

```diff
@@ -139,12 +132,10 @@ const Background = styled(Box)<{
   background-image: url(${props => props.headerImageUrl});
   background-size: cover;
   background-position: center;
-  ${props =>
-    props.isMobile &&
-    `
-      margin-left: -20px;
-      margin-right: -20px;
-  `};
+  ${media.xs} {
+    margin-left: -20px;
+    margin-right: -20px;
+  }
```

Comment:
I don't see a corresponding change in usage go this component above, is that correct?

Reply:
I didn’t pay attention and only now notice the media query `media.xs` 🙈